x/ibc/07-tendermint/types/client_state.go

```diff
@@ -30,54 +30,45 @@ type ClientState struct {
 	TrustingPeriod time.Duration `json:"trusting_period" yaml:"trusting_period"`
```

Comment:
I believe we should also have a variable TrustLevel field in the ClientState

Reply:
Yes, I agree, it can be configurable actually ("Poor man's FlexibleBFT")

Reply:
Let's split this to a separate PR though - https://github.com/cosmos/cosmos-sdk/issues/5686.